corehq/blobs/migrate.py

```diff
@@ -517,7 +517,7 @@ def __init__(self, slug, doc_types, doc_migrator_class):
         sorted_types = sorted(doc_type_tuples_to_dict(self.doc_types))
         self.iteration_key = "{}-blob-migration/{}".format(self.slug, " ".join(sorted_types))
 
-    def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100):
+    def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100, **options):
```

Comment:
I think c8b161b is incomplete. I was expecting `**options` to be removed here, and also the `**options` added to `ExportByDomain`.